basic-assembly/basic-cli/pom.xml

```diff
@@ -15,11 +15,11 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.eclipse.sw360.antenna</groupId>
-        <artifactId>antenna-basic-assembly</artifactId>
+        <artifactId>basic-assembly</artifactId>
         <version>${revision}${qualifier}</version>
     </parent>
 
-    <artifactId>antenna-cli</artifactId>
+    <artifactId>basic-cli</artifactId>
 
```

Comment:
I would prefer simply cli, basic implies that it is only rudimentary functionality, but it provides all features of Antenna, doesn't it?

Reply:
I thought the basic assembly is a prototype which can be adopted for internal use. And for that it is a basic version of an actual cli implementation.

Reply:
From my point of view, basic assembly should provide the possbility of using Antenna as is for dependency tracking and management. It should be possible to extend it, but it should not be necessary. Therefore the frontends of Antenna are fully usable frontends, at least they should be. I thought about basic-configuration, but this is "basic" since it is the fundamental configuration to get the system running. It has to be adapted e.g. with the right analysis tool for a certain use case. The question is about basic-assembly, where I think we should really target to have an assembly for Antenna. But this is only a grouping name, so it is not that important.